### PR TITLE
Update links and add spacing

### PR DESCRIPTION
**Pull request by ShalokShalom**
Changed a few things, moved some parts, improved readability. 

Updated the vscode plugin link and mention Elm in Elm once more, add the roadmap on top and introduced Elm like spacing in order to improve the readability of the source.

The line "In case of any questions about the project feel free to submit them in Issues with Q&A label" could create a new issue for the user - with q&a label already applied. Has to be done by a privileged person. 

Oh, and most important: Adds emoticons. :heart:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,99 +118,128 @@ cd elchemy
...
-- Elchemy-effects - **20%** - You can't and shouldn't write anything with side-effects in Elchemy yet. We're working on finding the best solution for effects that would fit both Elm's and Elixir's community (see [#297](https://github.com/wende/elchemy/issues/297) for more info)
-- Elchemy-core for Erlang VM - **5%** - Everything for os related tasks like filesystem, OTP goodies etc are yet to be done
-- Elchemy type checker - **15%** - Self-hosted elchemy type inference algorithm
 
 # Contributor credits:
 
 - Tomasz Cichociński - [@baransu](https://github.com/baransu)
 - Colin Bankier - [@colinbankier](https://github.com/colinbankier)
 - Nathaniel Knight - [@neganp](https://github.com/neganp)
+- [Elm-in-Elm](https://github.com/elm-in-elm/compiler) creator Martin Janiczek - [@janiczek](https://github.com/Janiczek)
```
Comment (wende) [inline README.md]: We don't use elm-in-elm at all. But it definitely is a good place to mention @Bogdanp for creating elm-ast which was the first dependency and the most influential inspiration in creating Elchemy

**Review by wende — CHANGES_REQUESTED** [review]
Looks good 🎉 
Just one comment

File: README.md (hunk 1 above)
Comment (wende) [reply README.md]: Also, feel free to mention yourself @ShalokShalom as recently you've been giving Elchemy more love than I was able to deliver 😄 

OT: I've been rewriting type inference from scratch and there's been some great developments in term of having it clear and concise. Really stoked to get this to MVP stage 🎉

**Comment by ShalokShalom** [comment]
Oh, I see

I have a talent to misread and interpret Slack chat conversations and Github issues, no idea where I got the feeling elm-in-elm is used.

So elm-ast is inspiration to your own implemenation - and why is the code for that one not public?

I think it would help showing that the project is still going on.

Your are doing great work!

P.S: Thanks for your motivation, I will add my name once I deliver the documentation for complete newbie which I talked about like a year ago. Or two.

**Comment by ShalokShalom** [comment]
Oh, I do see you now maintain the elm-ast repo

I guess there is the code, you are talking about?

**Comment by ShalokShalom** [comment]
@wende i added new changes, did you already look at them?

I have made an own section for people and projects who inpired your work, and seperated it from the list of contributors.

**Pull request merged**